### DISCUSSION TITLE
Please add Mistoline control net support.

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [Acly/krita-ai-diffusion#701] [Ideas]
Mistoline new kid on the scene. Very strict controlnet model. I am using my ComfyUI workflows. Please add this to Krita Diffusion. 
https://github.com/TheMistoAI/MistoLine

**Comment by Acly** [comment]
It looks like a good replacement for sai-sketch, but would be nice to have the "Anyline" preprocessor they mention in the paper too. It's not released yet I think.

**Reply by kubilaykilinc** [reply]
https://github.com/TheMistoAI/ComfyUI-Anyline

**Comment by syl1130** [comment]
To add Anyline and Mistoline ControlNet support to Krita AI, ComfyUI Control Net's Anyline offers seamless integration, high accuracy, and user-friendly design, enhancing control and efficiency in applications. Please add Anyline and Mistoline control net support.

**Reply by SyamsQ** [reply]
up
